src/test/java/org/elasticsearch/gateway/local/RecoveryBackwardsCompatibilityTests.java

```diff
@@ -77,7 +77,8 @@ public void testReusePeerRecovery() throws Exception {
         indexRandom(true, builders);
         ensureGreen();
         if (randomBoolean()) { // just make sure it doesn't break anything - we seal before we actually bump replicas
-            backwardsCluster().internalCluster().client().admin().indices().prepareSealIndices("test").get();
+            logger.info("--> trying to sync flush");
+            assertEquals(SyncedFlushUtil.attemptSyncedFlush(internalCluster(), "test").failedShards(), 0);
```

Comment:
I will add that, also to indexRandom() in ElasticsearchIntegrationTest. However, after running the test I found that the actual problem is that we check for no failures here - which is not good because the check is doomed to fail if a shard was an older version node. will remove the assert as well.